### PR TITLE
Workout controller tests for new and create

### PR DESCRIPTION
**Pull request by josephferrairo**
I added tests for the create and new action in workout, as well as modified the database_cleaner.rb file so that it cleans the database after every test is ran, this will ensure that we will not be getting duplicate errors while testing.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -274,4 +274,4 @@ DEPENDENCIES
   web-console
 
 BUNDLED WITH
-   1.12.5
+   1.13.1
```
Comment (BGRicker) [inline Gemfile.lock]: what's this for?

File: spec/fabricators/user_fabricator.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 Fabricator(:user) do
-  email                 "Arnold@Arnold.edu"
+  email                 "Arnold@Arnold.Gov"
```
Comment (BGRicker) [inline spec/fabricators/user_fabricator.rb]: why did you do this?

Comment (josephferrairo) [inline spec/fabricators/user_fabricator.rb]: i was going back and forth between using fabricator and not

Comment (BGRicker) [inline spec/fabricators/user_fabricator.rb]: the email is already in there but you just changed the email address from edu

**Pull request closed without merging**